rules/travis.js

```diff
@@ -74,15 +75,23 @@ module.exports = ctx => {
 				});
 			}
 
-			const versions = Array.isArray(travis.node_js) ? travis.node_js : [travis.node_js];
+			let versions = Array.isArray(travis.node_js) ? travis.node_js : [travis.node_js];
+
+			versions = versions.filter(v => IGNORED_VERSIONS.indexOf(v) === -1);
 
 			for (const version of versions) {
 				if (DEPRECATED_VERSIONS.indexOf(version) !== -1) {
 					ctx.report({
 						message: `Version \`${version}\` is deprecated.`,
 						file
 					});
-				} else if (version !== 'node' && engine && !semver.satisfies(normalize(version), engine)) {
+				}
+			}
+
+			versions = versions.filter(v => DEPRECATED_VERSIONS.indexOf(v) === -1);
```

Comment:
What do you think about merging them all into one filter?

```js
let versions = Array.isArray(travis.node_js) ? travis.node_js : [travis.node_js];

versions = versions.filter(version => {
  if (IGNORED_VERSIONS.indexOf(version) !== -1) {
    return false;
  } else if (DEPRECATED_VERSIONS.indexOf(version) !== -1) {
    ctx.report({
      message: `Version \`${version}\` is deprecated.`,
      file
    });

    return false;
  } else if (engine && !semver.satisfies(normalize(version), engine)) {
    ctx.report({
      message: `Unsupported version \`${version}\` is being tested.`,
      fix: fixers.unsupported(version),
      file
    });
  }

  return true;
});
```

This is what I initially tried to avoid but, on second thought, it might not be so bad.